frontend/scss/components/templates/docs-guides-overview.scss

```diff
@@ -47,36 +51,21 @@
     h1, h2 {
       margin-bottom: .5em;
     }
+  }
 
-    @media (min-width: 930px) {
-      .ap-m-breadcrumbs {
-        margin-left: 60px;
+  section {
+    &.#{utility('content')} {
+      padding: 62px 15px;
+      background-color: color('white');
+      box-shadow: 0 25px 40px 0 rgba(0,0,0,0.05);
```

Comment:
Either use color variable or check for a shadow mixin.

Reply:
Done